src/components/expandableSection/index.tsx

```diff
@@ -0,0 +1,43 @@
+import _ from 'lodash';
+import React from 'react';
+import {LayoutAnimation, StyleSheet} from 'react-native';
+import View from '../view';
+
+export type ExpandableSectionProps = {
+  /**
+   * expandableSection header element
+   */
+  sectionHeader?: JSX.Element;
+  /**
+   * expandableSection expandable children
+   */
+  children?: React.ReactNode;
+  /**
+   * expandableSection icon color
```

Comment:
You forgot changing the prop description (:

Reply:
🙊